CONTRIBUTING.md

```diff
@@ -25,50 +26,50 @@ examples.
 ### Submitting Pull Requests
 
 Any proposed changes to the specification should be done as pull requests. Please make these
-requests against the [`dev`](https://github.com/radiantearth/stac-api-spec/tree/dev) branch (this will
+requests against the [dev](https://github.com/radiantearth/stac-api-spec/tree/dev) branch (this will
 require you to switch from the default of 'master', which we keep so it displays first). 
 
 Creating a Pull Request will show our PR template, which includes checkbox reminders for a number
-of things.
-
-- Adding an entry the [CHANGELOG](CHANGELOG.md). If the change is more editorial and minor then this 
-is not required, but any change to the actual specification should definitely have one.
-- Base the PR against dev, as mentioned above - even if the branch was made off of dev this reminds
-you to change the base in GitHub's PR creation page.
-- Highlight if the PR makes breaking changes to the specification (in beta there can still be
-select breaking changes, but after 1.0 this will change)
+of things, including adding an entry the [CHANGELOG](CHANGELOG.md) and making the PR against the `dev`
+branch. 
 
 All pull requests should submit clean markdown, which is checked by the continuous integration
-system. Please use `npm run check-markdown` locally, as described in the [next section](#using-check-markdown-locally), 
+system. Please use `npm run check` locally, as described in the [next section](#check-files), 
 to ensure that the checks on the pull request succeed. If it does not then you can look at the
-mistakes online, which are the same as running `check-markdown` locally would surface.
+mistakes online, which are the same as running `npm run check` locally would surface.
 
-All pull requests that modify or create JSON examples should use [JSON 
-formatter](https://jsonformatter.org/) to keep files consistent across the repo. 
+All pull requests that modify or create JSON schema files or examples should use
+[JSON formatter](https://jsonformatter.org/) to keep files consistent across the repo. 
 
 All pull requests additionally require a review of two STAC core team members. Releases are cut
 from dev to master (and require 3 approvals).
 
-### Using check-markdown locally
+### Check files
 
-The same check-markdown program that runs as a check on PR's is part of the repo and can be run locally. 
-To install you'll need npm, which is a standard part of any [node.js installation](https://nodejs.org/en/download/). 
-Alternatively, you can also use [yarn](https://yarnpkg.com/) instead of npm. In this case replace all occurrences of 
-`npm` with `yarn` below.
+The same check-markdown and check-examples programs that runs as a check on PR's is part of the repo and can be run locally. 
```

Comment:
I don't think 'check-examples' is part of the api spec repo? Running 'check' seems to use check-markdown and check-openapi. Should update this to explain what check-openapi does. Running it right now seems to raise some warnings, so we should probably try to fix those:

```
/Users/cholmes/Repos/stac-api-spec/fragments/filter/cql.yml
 16:9  warning  oas3-parameter-description  Parameter objects should have a `description`.  components.schemas.booleanValueExpression.oneOf[3]
/Users/cholmes/Repos/stac-api-spec/fragments/filter/openapi.yaml
  10:9  warning  operation-description  Operation `description` must be present and non-empty string.  paths./.get
  10:9  warning  operation-tags         Operation should have non-empty `tags` array.                  paths./.get
 29:11  warning  operation-tag-defined  Operation tags should be defined in global tags.               paths./queryables.get.tags[0]
  42:9  warning  operation-description  Operation `description` must be present and non-empty string.  paths./collections/{collectionId}.get
  42:9  warning  operation-tags         Operation should have non-empty `tags` array.                  paths./collections/{collectionId}.get
 74:11  warning  operation-tag-defined  Operation tags should be defined in global tags.               paths./collections/{collectionId}/queryables.get.tags[0]
```